### DISCUSSION TITLE
How to get all the songs in the playlist?

### DISCUSSION THREAD

**Discussion by wenbin1354** [eritislami/evobot#1336] [Q&A] [answered]
Seem like the playlist only gets the first 100 songs in the playlist, changing the "MAX_PLAYLIST_SIZE" over 100 won't affect this, it still gets the first 100 songs in the playlist. 
How would I make it so that this playlist gets all the songs in a playlist that has over 100 songs?
I'm passing in a youtube playlist, use this playlist that has over 100 songs, for example, 
https://www.youtube.com/watch?v=bu7nU9Mhpyo&list=PLcobBUUBYD76n5OKAe_CFxkwTJXaW3-It

**Comment by eritislami** [comment, marked as answer]
Basic
```js
YouTube.getPlaylist("some_youtube_playlist")
  .then(console.log) // max 100 items
  .catch(console.error);
```
Fetch all videos immediately
```js
YouTube.getPlaylist("some_youtube_playlist", { fetchAll: true })
  .then(console.log)
  .catch(console.error);
```
https://www.npmjs.com/package/youtube-sr
